frontend/Dockerfile.dev

```diff
@@ -6,4 +6,4 @@ COPY package*.json ./
 
 RUN npm install
 
-COPY . /app/
+# COPY . /app/
```

Comment:
Why is this commented?

Reply:
Forgot to remove the line
Fixed in [709f3e6](https://github.com/SELab-2/UGent-7/pull/372/commits/709f3e600e9e76fd79375c10c7cbbf15222c0625)